about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Astyle.css">
+    <title>LEADING DESIGN FESTIVAL</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <button type = "submit" class="hamburger">
+        <span></span>
+        <span></span>
+        <span></span>
+      </button>
+      <div class="dNav">
+        <a class = "brand" href="index.html"><img src="./Logo/ld.png" alt=""> Leading Design Festival</a>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="">Program</a></li>
+          <li><a href="">Join</a></li>
+          <li><a href="">Sponsor</a></li>
+        </ul>
+        <a class="Campaign" href="#">LD Campaign</a>
+      </div>
+    </nav>
+  </header>
+  <section class="intro">
+    <span class="hello">"hello Sharing world"</span>
+    <h1>LEADING DESIGN FESTIVAL GLOBAL SUMMIT 2021</h1>
+    <p class="description">the global summit brings together thecommunity of experts, academics, and activists who comprise the creative commonns affilate network in a differrent country every two.
+      this year, we hope to expand our invittaion list including organizations and individuels
+    </p>
+    <p>please contact us peremail foranyfuture questions
+      about LD Festival summit 2021
+    </p>
+    <a href="gmail.com">anas.chaabane98@gmail.com</a>
+    <p class="broughtBy">@Clearleft.</p>
+  </section>
+  <section class="about">
+    <h4>Leading Design Festival Summit 2021 Logo</h4>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi esse obcaecati corporis accusamus atque cumque, ab quam aspernatur reprehenderit veniam? Velit quod, itaque consequatur dolore sequi fugiat a rerum ad mollitia, suscipit tenetur? Officiis minima excepturi numquam corrupti mollitia dolor.</p>
+    <a class = "Logo" href="index.html"><img src="./Logo/ld.png" alt="Logo"><h2>Leading Design Festival</h2> </a>
+  </section>
+  <section class="footer">
+   <h4> See the past LD Festival Summit</h4>
+   <div class="imgs">
+   <div class="img"><img src="./assest/home-hero_f5cf215b1a5f05c503d3c7ec2c0d70ce.jpg" alt="">
+   <div class="overlay">
+    <h4>2019</h4>
+    <p> LD Global summit 2019 in tokyo </p>
+   </div>
```

Comment:
- [ ] Kindly add this orange(#EC5242) short line bar on the About Page after the h4 title and don't forget to add the missing description paragraph after the title 👇🏻 :sunglasses::nerd_face:
-  I know you can do it :muscle: 😄 

- Your Design:

![30about](https://user-images.githubusercontent.com/79058364/176781878-f97d8693-7af8-41c1-bf27-c1e269ee242a.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![30aboutcool](https://user-images.githubusercontent.com/79058364/176781847-72dee75f-29c5-41a6-8496-16bed5896968.png)